fhirest-test-app/src/test/java/ee/fhir/fhirest/core/service/conformance/HapiContextHolderCacheWithConfiguredEnvTest.java

```diff
@@ -0,0 +1,44 @@
+package ee.fhir.fhirest.core.service.conformance;
+
+import org.hl7.fhir.common.hapi.validation.support.ValidationSupportChain;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.util.ReflectionTestUtils;
+
+import java.util.List;
+
+import static org.hl7.fhir.common.hapi.validation.support.ValidationSupportChain.CacheConfiguration;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@SpringBootTest
+@TestPropertySource(properties = {"CACHE_TIMEOUT=2"})
+public class HapiContextHolderCacheWithConfiguredEnvTest {
```

Comment:
separate classes for two usecases seems too much. ideally it should be HapiContextHolderTest with two tests. it should be possible to overwrite proerties inline.   
although, to be honest, i feel like testing hapi and spring is not really necessary at all